operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -313,17 +313,20 @@ func (r *ReconcileIstioOperator) Reconcile(_ context.Context, request reconcile.
 	if err != nil {
 		return reconcile.Result{}, nil
 	}
-	var jwtPolicy util.JWTPolicy
-	if jwtPolicy, err = util.DetectSupportedJWTPolicy(kubeClient); err != nil {
-		// TODO(howardjohn): add to dictionary. When resolved, replace this sentence with Done or WontFix - if WontFix, add reason.
-		scope.Warnf("Failed to detect third-party JWT support: %v", err)
-	} else {
-		if jwtPolicy == util.FirstPartyJWT {
-			scope.Info("Detected that your cluster does not support third party JWT authentication. " +
-				"Falling back to less secure first party JWT. " +
-				"See " + url.ConfigureSAToken + " for details.")
+	if iopMerged.Status == nil || iopMerged.Status.Status == v1alpha1.InstallStatus_NONE {
```

Comment:
I am not sure how this works? globalValues is just a local variable, it will not mutate the spec in the cluster (?). This means it will not actually be updated? So on the next reconcile we will have the wrong value I would think